id/guide/installation.md

```diff
@@ -79,46 +104,46 @@ Proyek memerlukan file `package.json` untuk memperinci bagaimana untuk memulai `
 }
 ```
 
-`scripts` akan menjalankan Nuxt.js melalui `npm run dev`.
+`scripts` akan menjalankan Nuxt.js dengan `npm run dev`.
 
-### Memasang `nuxt`
+### Instalasi `nuxt`
 
-Setelah `package.json` selesai dibuat, tambahkan `nuxt` ke dalam proyek melalui npm:
+Setelah file `package.json` dibuat, tambahkan `nuxt` pada projek anda dengan npm:
 
 ```bash
 npm install --save nuxt
 ```
 
 ### Direktori `pages`
 
-Nuxt.js akan mengubah setiap file `*.vue` di dalam direktori `pages` sebagai rute untuk aplikasi.
+Nuxt.js akan men-transformasi setiap file `*.vue` yang berada dalam direktori `pages` sebagai route pada aplikasi anda
 
-Buat direktori `pages`:
+Membuat direktori `pages`:
 
 ```bash
 $ mkdir pages
 ```
 
-lalu buat halaman pertama dalam `pages/index.vue`:
+kemudian buat halaman pertama anda pada `pages/index.vue`:
 
 ```html
 <template>
   <h1>Hello world!</h1>
 </template>
 ```
 
-dan jalankan proyek dengan:
+dan jalankan projek dengan:
 
 ```bash
 $ npm run dev
 ```
 
-Kini aplikasi berjalan pada http://localhost:3000.
+Aplikasi sekarang berjalan pada on http://localhost:3000.
 
 <div class="Alert">
 
-Nuxt.js akan memantau perubahan file di dalam direktori <code>pages</code>, jadi tidak perlu me-restart aplikasi saat menambahkan halaman baru.
+Nuxt.js akan memantau perubahan pada file anda yang berada dalam direktori <code>pages</code>, jadi anda tidak perlu menjalankan ulang aplikasi anda ketika menambahkan halaman baru.
 
 </div>
 
-Untuk mengetahui lebih lanjut tentang struktur direktori pada proyek: [Dokumentasi Struktur Direktori](/guide/directory-structure).
+Untuk menjelajahi lebih lanjut mengenai struktur direktori pada projek : [Directory Structure Documentation](/guide/directory-structure).
```

Comment:
I think the proper translation for `to discover more about` is 'untuk menemukan lebih banyak' not 'menjelajahi lebih lanjut'

Reply:
thank you for your review, i've updated the translations